ams/ams-server/src/main/java/com/netease/arctic/ams/server/config/ArcticMetaStoreConf.java

```diff
@@ -138,7 +138,7 @@ public class ArcticMetaStoreConf {
   public static final ConfigOption<Long> TABLE_FILE_INFO_CACHE_INTERVAL =
       ConfigOptions.key("table.file.info.cache.interval")
           .longType()
-          .defaultValue(6 * 60 * 60 * 1000L)
+          .defaultValue(5 * 60 * 1000L)
```

Comment:
why do us change this default value from 6 hours to 5 minutes?

Reply:
For a more timely update to file info. Because minor optimize is frequent.